### PR TITLE
Feature/Add sink connector

### PR DESCRIPTION
**Pull request by episanchez**

### CODE REVIEW

File: src/main/java/com/oyst/kafka/connect/nats/sink/NatsSinkConnectorConfig.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+   * Transform process properties.
+   *
+   * @param properties associative array with properties to be process
+   */
+  public NatsSinkConnectorConfig(Map<String, String> properties) {
+    super(config, properties);
+    LOG.info("Initialize transform process properties");
+  }
+
+  public static void main(String[] args) {
```
Comment (damiencarol) [inline src/main/java/com/oyst/kafka/connect/nats/sink/NatsSinkConnectorConfig.java:61]: I don't think it is ok to have it.

**Review by damiencarol — CHANGES_REQUESTED** [review]

File: src/main/java/com/oyst/kafka/connect/nats/sink/NatsSinkConnectorConfig.java (hunk 1 above)
Comment (episanchez) [reply src/main/java/com/oyst/kafka/connect/nats/sink/NatsSinkConnectorConfig.java:61]: Ok I change that

**Review by episanchez — COMMENTED** [review]
Ok

File: pom.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.oyst</groupId>
     <artifactId>kafka-connect-nats</artifactId>
-    <version>0.2</version>
+    <version>0.3.0</version>
```
Comment (damiencarol) [inline pom.xml:9]: :+1:

**Review by damiencarol — APPROVED** [review]

**Pull request merged**